agent/roles/manifests/templates/install-config_baremetal_yaml.j2

```diff
@@ -21,6 +21,20 @@ controlPlane:
   hyperthreading: Enabled
   name: master
   replicas: {{ num_masters }} 
+{% if num_arbiters == "0" and num_masters == "2" %}
+{% set hostnames = agent_nodes_hostnames.split(',') %}
```

Comment:
Also below the same code for extracting hostnames and bmc details is used, as a minor point it could be the occasion to refactor it and remove the duplication (for the jinja vars definition)

Reply:
That's a great point, wasn't too familiar with jinja var scoping, but I went ahead and moved the duplicated instantiated vars at the top of the file, is that what you had in mind for this?